l10n_it_ricevute_bancarie/__init__.py

```diff
@@ -24,5 +24,4 @@
 #
```

Comment:
Change header with this model: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1